src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAXIMUM_TASKS = 100;
+    private static final String DIVIDER = "===================================================";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        printWelcomeMessage();
+        Scanner sc = new Scanner(System.in);
+        String name = sc.nextLine();
+        printUserGreeting(name);
+        manageTasks(sc, name);
+    }
+
+    private static void manageTasks(Scanner sc, String name) {
+        Task[] tasks = new Task[MAXIMUM_TASKS];
+        int taskNumber = 0;
+
+        while (true) {
+            String input = sc.nextLine();
+            String[] parseInput = input.split(" ",2);
+            String command = parseInput[0];
+
+            if (command.equalsIgnoreCase("done")) {
```

Comment:
Could maybe consider using switch statements as it will make the code more readable.